data/json/items/gun/44.json

```diff
@@ -1,5 +1,34 @@
 [
   {
+    "id": "cane_rifle",
+    "type": "GUN",
+    "name": "walking cane",
+    "description": "Gun in a cane.  A custom-built .44 Magnum rifle carefully integrated into a walking cane.  It's also reinforced for hand-to-hand combat.  Never underestimate a handicapped person.",
+    "weight": 950,
+    "volume": 4,
+    "price": 400000,
+    "bashing": 15,
+    "to_hit": 2,
+    "material": [ "steel", "aluminum", "wood" ],
+    "symbol": "/",
+    "color": "dark_gray",
+    "skill": "rifle",
+    "ammo": "44",
+    "ranged_damage": 11,
+    "dispersion": 100,
+    "sight_dispersion": 75,
+    "aim_speed": 7,
+    "recoil": 90,
+    "durability": 8,
+    "loudness_modifier": 25,
+    "clip_size": 1,
+    "reload": 120,
```

Comment:
The cane rifle is a single-shot weapon. The reload speed will depend on the type of action. At the moment I think a special kind of bolt action would be best.